src/app.ts

```diff
@@ -20,9 +20,20 @@ app.post('/calculateRaw', async (req, res) => {
       patients,
       options || {} // options are optional, so this defaults to an empty Object
     );
-    return res.json(rawResult.results);
+    /**
+     * TODO:
+     * Format of RawResults output should maybe be changed in fqm-execution to adapt to new withErrors field.
+     * Currently rawResults output is either the rawResults object (on a successful run) or an error message
+     * with an explanation for the failure. I think we should instead just throw an error upon failure, not
+     * return an error string.
+     */
+    return res.json({ results: rawResult.results, warnings: rawResult.withErrors });
```

Comment:
Please checkout above comment. lmk if you disagree!

Reply:
I agree